docs/UserGuide.md

```diff
@@ -3,14 +3,141 @@ layout: page
 title: User Guide
 ---
 
+# Table of Contents
+<!-- TOC -->
+* [Introduction](#introduction)
+  * [About TaskHub](#about-taskhub)
+  * [About this User Guide](#about-this-user-guide)
+  * [Navigating the User Guide](#navigating-the-user-guide)
+* [Getting Started](#getting-started)
+  * [Installing and launching TaskHub](#installing-and-launching-taskhub)
+  * [Understanding the components of TaskHub](#understanding-the-components-of-taskhub)
+  * [Understanding the User Interface(UI)](#understanding-the-user-interface--ui-)
+  * [Quick start](#quick-start)
+* [Features](#features)
+  * [General Commands](#general-commands)
+    * [View help : `help`](#view-help--help)
+      * [GUI Help:](#gui-help-)
```

Comment:
Seems like there is an extra colon after `[GUI Help`! Also, the link to the section seems to not be working.

Reply:
Yeah, this part was supposed to be deleted, good catch.